devtools/gradle/src/test/resources/multi-module-project/settings.gradle

```diff
@@ -0,0 +1,13 @@
+pluginManagement {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    plugins {
+        id 'io.quarkus' version '999-SNAPSHOT'
```

Comment:
We can't be using specific quarkus versions in the test configs. This will break the release process.